ceph-releases/hammer/ubuntu/14.04/daemon/config.kv.sh

```diff
@@ -51,13 +51,13 @@ function get_mon_config {
     kviator --kvstore=${KV_TYPE} --client=${KV_IP}:${KV_PORT} ${KV_TLS} get ${CLUSTER_PATH}/bootstrapRgwKeyring > /var/lib/ceph/bootstrap-rgw/${CLUSTER}.keyring
 
 
-    if [ ! -f /etc/ceph/monmap ]; then
+    if [ ! -f /etc/ceph/monmap-${CLUSTER} ]; then
       echo "Monmap is missing. Adding initial monmap..."
-      kviator --kvstore=${KV_TYPE} --client=${KV_IP}:${KV_PORT} ${KV_TLS} get ${CLUSTER_PATH}/monmap > /etc/ceph/monmap
+      kviator --kvstore=${KV_TYPE} --client=${KV_IP}:${KV_PORT} ${KV_TLS} get ${CLUSTER_PATH}/monmap-${CLUSTER} > /etc/ceph/monmap-${CLUSTER}
```

Comment:
actually I don't think you need to modify the key here. CLUSTER is part of CLUSTER_PATH